crates/forge_analyzer/src/definitions.rs

```diff
@@ -2466,6 +2478,26 @@ impl Visit for FunctionCollector<'_> {
                     }
                 }
             }
+            Some(Expr::Object(object_lit)) => {
+                object_lit.props.iter().for_each(|prop| {
+                    if let PropOrSpread::Prop(prop) = prop {
+                        if let Prop::Method(MethodProp { key, function }) = &**prop {
+                            if let Some(key_ident) = key.as_ident() {
+                                if let Some(def_id_function) =
+                                    self.res
+                                        .get_prop_from_ident(self.module, id.clone(), key_ident)
```

Comment:
This clone is unnecessary, since `DefId` is `Copy`. You can dereference it instead: `*id`.

Reply:
I needed to add the to_id() function instead. The dereferencing didn't work.

Reply:
Hope this works.